joint_limits:
     - joint: "shoulder_pan"
       type: "revolute"
       upper_limit: 0.7
       lower_limit: -1.5

     - joint: "shoulder_lift"
       type: "continuous"
       upper_limit: 3.1415
       lower_limit: -3.1415